src/test/java/com/exasol/adapter/databricks/PushdownQueryIT.java

```diff
@@ -17,9 +17,9 @@
 class PushdownQueryIT extends AbstractIntegrationTestBase {
 
     @TestFactory
-    Stream<DynamicNode> selectlistFilterOrderLimit() {
+    Stream<DynamicNode> selectListFilterOrderLimit() {
         return testSetup
-                .pushdownTest(databricksSchema -> databricksSchema.createTable("tab", "ID", "INT", "NAME", "STRING")
+                .pushdownTest(databricksSchema -> databricksSchema.createTable("tab", "id", "INT", "name", "STRING")
```

Comment:
Intentionally use lower case table & column in databricks to verify, that pushdown query contains lowercase names.